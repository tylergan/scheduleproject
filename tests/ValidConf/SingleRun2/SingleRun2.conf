on Tuesday at 1100 run /usr/bin/python3 tests/ValidConf/SingleRun2/SingleRun2.py