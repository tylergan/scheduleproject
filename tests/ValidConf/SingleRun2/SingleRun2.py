print("Adding 1 + 1: The answer is {:.2f}".format(1+1))