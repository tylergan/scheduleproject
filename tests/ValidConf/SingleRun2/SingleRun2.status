will run at XX XX XX 11:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun2/SingleRun2.py
ran XX XX XX 11:00:00 2020 /usr/bin/python3 tests/ValidConf/SingleRun2/SingleRun2.py
