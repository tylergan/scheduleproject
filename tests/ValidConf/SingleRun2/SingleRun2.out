Adding 1 + 1: The answer is 2.00
nothing left to run
